deployment/base/sa.yml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: jenkins
+automountServiceAccountToken: true
```

Comment:
Auto-mount else K8s plugin will throw error, and there is no pod agents capabilities